app/assets/javascripts/spotlight/application.js

```diff
@@ -3,6 +3,7 @@
 //= require sir-trevor
 //= require clipboard
 //= require tiny-slider
+//= require Sortable.min.js
```

Comment:
Is this only used by the admin interfaces? Is that why the nestable import was in [app/javascript/spotlight/admin/index.js](https://github.com/projectblacklight/spotlight/pull/3055/files#diff-45150080ff3f9be0628913e9f2fc161c5542df2eda3af97bafb429e235180621)?

Reply:
Yes, I believe that's the case. I'll shuffle this back.